README.md

```diff
@@ -95,6 +94,10 @@ We'll insert the following middlewares into the rails stack:
 2. `Rack::SslEnforcer`: enforces HTTPS.
 3. `Rack::Deflater`: compresses responses from the application, can be disabled with `ROO_ON_RAILS_RACK_DEFLATE` (default: 'YES').
 
+### Database configuration
+
+The database statement timeout will be set to a low value by default. Use `DATABASE_STATEMENT_TIMEOUT` (milliseconds, default 200) to customise.
```

Comment:
This is missing the note I added about it not doing anything in Rails 3. Should probably keep that.